feature/bgp/gracefulrestart/feature.textproto

```diff
@@ -56,6 +56,14 @@ telemetry_path {
   path: "/network-instances/network-instance/protocols/protocol/bgp/neighbors/neighbor/afi-safis/afi-safi/graceful-restart/state/received"
 }
 
+# Global telemetry paths
+config_path {
```

Comment:
Same here. Can we also update the Config Library code. We should strive to keep it in sync. We only have BGP base, BGP GR, NI code that needs to be updated.

Reply:
Done! Have added this as well to the config library code!